tests/fsharpqa/Source/Conformance/SpecialAttributesAndTypes/Imported/CallerInfo/CallerMemberName.fs

```diff
@@ -45,6 +47,8 @@ type MyTy() =
             |> List.head
         MyTy.Check(result, Some("CheckMembers"), "lambda")
         MyTy.Check(functionVal (), Some("functionVal"), "functionVal")
+        MyTy.Check(anonymousLambda, Some("anonymousLambda"), "anonymousLambda")
+        MyTy.Check(asyncVal, Some("asyncVal"), "asyncVal")
```

Comment:
It needs to be laid out in the RFC, but the behavior is that "top-level" bindings (methods, module-level and class-level functions, module-level and class-level values) will be captured. Sub-level bindings ("local variable" let bindings, nested functions) are not captured.

So for your example (we do have tests for this kind of thing), let's say it's part of a module:
```
module Test
let f () = 
    let x = 
        let g () = MyTy.GetCallerMemberName() 
        g()
    x
```

I would expect `f` to be the result.

I think one can defend all sorts of possible positions, but this was the most natural result when we coded it, and it seems to make sense to me. The name "member" hints at something primarily associated with the type or module. Is a local variable or a nested function a "member" of the type? I'd say no.